exercises/java-collections/src/main/java/com/cbfacademy/CollectionsExercises.java

```diff
@@ -4,26 +4,80 @@
 
 public class CollectionsExercises {
 
-    public LinkedList<Integer> useLinkedList() {
+
+
+    public static LinkedList<Integer> useLinkedList() {
         // TODO: create an empty linked list
         //  - add 4 as the first element of the list
         //  - then add 5, 6, 8, 2, 9 to the the list
         //  - add another 2 as the last element of the list
         //  - add 4 as the 3rd element of the list
         //  - invoke the method element() on the list and print the result on the screen
         //  - return the list
-        throw new RuntimeException("Not implemented");
+
+        LinkedList<Integer> newList = new LinkedList<>();
+        newList.addFirst(4); // 0
```

Comment:
You shouldn't need to use `addFirst` here, as the list is empty at this point. You should be able to use `add`